rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/rename/Common.rsc

```diff
@@ -55,6 +54,9 @@ data RenameConfig(
   , PathConfig(loc) getPathConfig = PathConfig(loc l) { throw "No path config for <l>"; }
 );
 
+data ChangeAnnotation = changeAnnotation(str label, str description, bool needsConfirmation = false);
+data TextEdit(ChangeAnnotation annotation = changeAnnotation("Rename", "Rename"));
```

Comment:
nice :+1:

Reply:
Indeed, much better and easier to use like this!